terraform/a2ha-terraform/modules/efs/main.tf

```diff
@@ -25,7 +25,7 @@ locals {
 resource "null_resource" "mount_efs" {
   count = length(local.ip_list)
   connection {
-    host        = "${ local.ip_list[count.index] }"
+    host        = " local.ip_list[count.index] "
```

Comment:
Adding quotes might consider the variable as string. 
Replace `" local.ip_list[count.index] "` with `local.ip_list[count.index]`

Reply:
fixed